endorser/requirements.txt

```diff
@@ -29,9 +29,9 @@ sniffio==1.2.0
 SQLAlchemy==1.4.27
 sqlalchemy2-stubs==0.0.2a19
 sqlmodel==0.0.6
-starlette==0.36.2
+starlette==0.35.1
```

Comment:
Downgrading this necessary to get it building?

Reply:
Unfortunately yes. I could try to upgrading the dependency requiring the downgrade.

Reply:
Ah yeah looks like FastAPI version restricts the starlette version